sdk/cosmosdb/cosmos/src/request/RequestOptions.ts

```diff
@@ -1,11 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { SharedOptions } from "./SharedOptions";
+import { DedicatedGatewayRequestOptions, SharedOptions } from "..";
 
 /**
  * Options that can be specified for a requested issued to the Azure Cosmos DB servers.=
  */
-export interface RequestOptions extends SharedOptions {
+export interface RequestOptions extends SharedOptions, DedicatedGatewayRequestOptions {
```

Comment:
From offline discussion: We have to also add it to `FeedOptions` to cover query operations

Reply:
fixed